Makefile

```diff
@@ -107,7 +107,7 @@ safety: ## Runs `safety check` to check python dependencies for vulnerabilities
 .PHONY: bandit
 bandit: ## Run bandit with medium level excluding test-related folders
 	pip install --upgrade pip && \
-        pip install --upgrade bandit!=1.6.0 && \
+        pip install --upgrade bandit==1.5.1 && \
```

Comment:
It looks like the behaviour of the --exclude argument changed in 1.6.1 - you now have to include the root directory in exclude paths, i.e. ``bandit --recursive parent --exclude parent/child`` as opposed to the previous ``bandit --recursive parent --exclude child``. This might be better than pinning to an older version.

Reply:
Omitted from this PR and rebased to `develop` as this was fixed in #4530.